experiments/generic/pod_cpu_hog/pod_cpu_hog_k8s_job.yml

```diff
@@ -43,7 +43,7 @@ spec:
             value: '1'
 
           - name: TOTAL_CHAOS_DURATION
-            value: '60000' # in ms
+            value: '60' 
```

Comment:
Add one comment `#in sec`

Reply:
By default we are taking sec to no need to mention it. If we wll take any other unit then we have to mention it.

Reply:
yeah, we can do that

Reply:
ok @uditgaurav !